compiler/qsc_rca/tests/assigns.rs

```diff
@@ -132,3 +132,102 @@ fn check_rca_for_dynamic_double_assign_to_local() {
         ],
     );
 }
+
+#[test]
+fn chec_rca_for_assign_call_result_to_tuple() {
```

Comment:
The test for call results are great! Do we also need some for a variable? As in:
```qsharp
let x = Foo();
mutable a = 1;
mutable b = 2;
set (a, b) = x;
```

Reply:
It wouldn't hurt to add this test, I'll add it!

Reply:
More tests added.